contracts/script/multisigTransactionProposals/safeSDK/modifyProverModeProposal.ts

```diff
@@ -102,6 +128,57 @@ function createPermissionedProverTxData(proverAddress: string): string {
   return data; // Return the encoded transaction data
 }
 
+/**
+ * Function to propose the transaction data for disabling permissioned prover mode
+ * @param {Safe} safeSDK - An instance of the Safe SDK
+ * @param {SafeApiKit} safeService - An instance of the Safe Service
+ * @param {string} signerAddress - The address of the address signing the transaction
+ * @param {string} safeAddress - The address of the Safe multisig wallet
+ */
+export async function proposeDisableProverTransaction(
+  safeSDK: Safe,
+  safeService: SafeApiKit,
+  signerAddress: string,
+  safeAddress: string,
+) {
+  // Prepare the transaction data to disable permissioned prover mode
+  let data = createDisablePermissionedProverTxData();
+
+  const contractAddress = getEnvVar("LIGHT_CLIENT_CONTRACT_ADDRESS");
+  isValidEthereumAddress(contractAddress);
+
+  // Create the Safe Transaction Object
+  const safeTransaction = await createSafeTransaction(safeSDK, contractAddress, data, "0");
```

Comment:
Due to the signature of createSafeTransaction, this doesn't indicate it's return type in intellisense.  By removing the explicit return type in `createSafeTransaction`, you can get some type inference here, that could prevent future errors due to the type checking alone.  Othwerise, this allows `createSafeTransaction` to change it's implementation with no errors indicated in the editor, or when the typescript gets interpretted.

Reply:
It looks like `SafeTransaction` isn't exposed directly in the `@safe-global/protocol-kit` package, which is saddening.  You **could** reach into the package to retrieve it via 
```typescript
import SafeTransaction from "@safe-global/protocol-kit/dist/src/utils/transactions/SafeTransaction";
```

But this seems rather rude.  A better solution might be to declare a local forward declaration of the type via the resolved return type of `safeSDK.createTransaction`:


```typescript
type SafeTransaction = Awaited<ReturnType<Safe['createTransaction']>>;
```

Reply:
Yea I initially considered creating a local type definition when I realized that it wasn't exposed but thought that it wasn't the best approach. I love this approach, thank you. implemented here https://github.com/EspressoSystems/espresso-sequencer/commit/158ed1bcf3d3ee310bbd185f2d6c90690c13472d